instrumentation/opentelemetry-instrumentation-fastapi/tests/test_fastapi_instrumentation.py

```diff
@@ -98,7 +98,7 @@ async def _(param: str):
             return {"message": param}
 
         @app.get("/healthzz")
-        async def health():
+        async def health():  # pylint: disable=unused-variable
```

Comment:
How did this even trigger pylint. Seems like the new version is very noisy

Reply:
This *is* an unused var actually but there is no reason why it can't be named `_` like the rest of the handlers defined inside this method. Updated.